PowerPointLabs/PowerPointLabs/TooltipsLab/CreateTooltip.cs

```diff
@@ -35,13 +36,15 @@ public static PowerPoint.Shape GenerateCalloutWithReferenceTriggerShape(PowerPoi
         {
             float midpointX = ShapeUtil.GetMidpointX(triggerShape);
 
-           
+
             PowerPoint.Shape callout = currentSlide.Shapes.AddShape(
                 Microsoft.Office.Core.MsoAutoShapeType.msoShapeRoundedRectangularCallout,
                 midpointX - TooltipsLabConstants.CalloutShapeDefaultWidth/2 + (float)(TooltipsLabConstants.CalloutArrowheadHorizontalAdjustment * TooltipsLabConstants.CalloutShapeDefaultWidth),
                 triggerShape.Top - (float)(TooltipsLabConstants.CalloutArrowheadVerticalAdjustment * TooltipsLabConstants.CalloutShapeDefaultHeight) - TooltipsLabConstants.TriggerShapeAndCalloutSpacing,
                 TooltipsLabConstants.CalloutShapeDefaultWidth,
                 TooltipsLabConstants.CalloutShapeDefaultHeight);
+            callout.TextEffect.Alignment = MsoTextEffectAlignment.msoTextEffectAlignmentCentered;
+            ShapeUtil.FormatCalloutToDefaultStyle(callout);
 
             callout.TextFrame2.AutoSize = MsoAutoSize.msoAutoSizeTextToFitShape;
             callout.TextEffect.Alignment = MsoTextEffectAlignment.msoTextEffectAlignmentCentered;
```

Comment:
Just saw this after I merged, sorry. Is line 50 a repeat of line 46?

Reply:
I think so (probably from merging the dev-release into this branch). Where should I remove this, in a PR just for this 1 line?